### DISCUSSION TITLE
lz4 dependency for rsync fails to install macOS Ventura 13.0.1

### DISCUSSION THREAD

**Discussion by MrLimoAK** [Homebrew/discussions#4085] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: >=2.5.0 (shallow or no git repository)
ORIGIN: https://github.com/Homebrew/brew
HEAD: e76c55e18434c870cca1de7aadf9d5218271fe2d
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 04f0013201ec455ac494a5e62dfa40cd7d89d208
Core tap last commit: 29 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 1.8.4 => /usr/local/bin/git
Curl: 7.84.0 => /usr/bin/curl
macOS: 13.0.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
```

### Output of `brew doctor`

```shell
brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: An outdated version (1.8.4) of Git was detected in your PATH.
Git 2.14.3 or newer is required for Homebrew.
Please upgrade:
  brew install git


---------------------------------
brew install git
==> Fetching dependencies for git: gettext and pcre2
==> Fetching gettext
==> Downloading https://ghcr.io/v2/homebrew/core/gettext/manifests/0.21.1
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/1c2f2c62faee672530e8e8e2695f99d26d1b606e74b289d1914dfa13c732c500--gettext-0.21.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gettext/blobs/sha256:fd7e48065cf73e37dfdf4c5cb789a14b93cf58ac06060814a60c94b87d8f26e6
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/a6a451ad6e18b9415a6139ca5cb4ced3a8d1e40c1238f913651d8100e6427452--gettext--0.21.1.ventura.bottle.tar.gz
==> Fetching pcre2
==> Downloading https://ghcr.io/v2/homebrew/core/pcre2/manifests/10.42
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/6a53794fcaabc5cc5e05b19c02ca9c4c5f2cb9a4d65a5790a6841146465b040f--pcre2-10.42.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/pcre2/blobs/sha256:7f414ed9d561dc85aacd41c7d18a452d3f58a6fe73af02b8fb876483080ec4df
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/75d2907d28236833dcb172288d16fe09964b80df40aeb49b044d6007a069b846--pcre2--10.42.ventura.bottle.tar.gz
==> Fetching git
==> Downloading https://ghcr.io/v2/homebrew/core/git/manifests/2.39.0
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/636072c249a3470615051e25b11bd19d0371cee8acc2673db9a6fb19d78c8215--git-2.39.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/git/blobs/sha256:f8e49ae6974d47ee1b7a3ed929ec5ff0524cca3f6bb92328a478a72019e7b24f
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/9153e522b9998391e74b52a7dca6ea130ae5d78331752aca770680c99444f354--git--2.39.0.ventura.bottle.tar.gz
==> Installing dependencies for git: gettext and pcre2
==> Installing git dependency: gettext
Unknown option: -C
usage: git [--version] [--help] [-c name=value]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
Error: Command failed with exit 129: git
```

### Description of issue

$ brew install lz4
==> Fetching lz4
==> Downloading https://ghcr.io/v2/homebrew/core/lz4/manifests/1.9.4
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/379e59b981667f9585b33a2ff318769d8edca3ce6fd2e9a67ed291ae3e0cc872--lz4-1.9.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/lz4/blobs/sha256:6a911ee2a3ea072f414d2983d532b28c34b63a68ff388a0008e1528dc0668838
Already downloaded: /Users/.../Library/Caches/Homebrew/downloads/0f1d1abbda9632250a53d7d8bb7709344db6249bff1f30759a69848549aaca4f--lz4--1.9.4.ventura.bottle.tar.gz
Unknown option: -C
usage: git [--version] [--help] [-c name=value]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
Error: Command failed with exit 129: git

### Relevant casks

rsync git

**Comment by gromgit** [comment]
What's the output of:
```
/usr/bin/git --version
ls -l /usr/local/bin/git
```
?

**Comment by MrLimoAK** [comment]
/usr/bin/git --version
git version 2.37.1 (Apple Git-137.1)

ls -l /usr/local/bin/git
lrwxr-xr-x  1 root  wheel  66 Jan  3  2014 /usr/local/bin/git ->
/Users/.../Applications/GitHub.app/Contents/Resources/git/bin/git

On Thu, Jan 5, 2023 at 5:16 PM Adrian Ho ***@***.***> wrote:

> What's the output of:
>
> /usr/bin/git --version
> ls -l /usr/local/bin/git
>
> ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Homebrew/discussions/4085#discussioncomment-4607564>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABQFVQGVWPVKUTKCOQQ5DU3WQ557DANCNFSM6AAAAAATSJ53OY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
       \\\|///
     \\  - -  //
      (  @ @  )
+---oOOo-(_)-oOOo--------------------+
| Greg Schiedler - KL3MO
| Limo.Net Internet Services
| PO Box 167
| Bethel, AK 99559-0167
| 541.990.7911/800.767.3932
| Fax: 888.777.6120
|   ooo0         http://www.Limo.Net
|  (   )    0ooo E-Mail: ***@***.***
+---\ (-----(   )--------------------+
     \_)     ) /
            (_/

**Reply by SMillerDev** [reply]
Try updating the GitHub app
